projects/ngrx.io/content/guide/component-store/effect.md

```diff
@@ -91,3 +91,24 @@ An easy way to handle the response in ComponentStore effects in a safe way, with
     );
   });
 </code-example>
+
+## Calling an `effect` without parameters
+
+A common use case is to call the `effect` method without any parameters. 
+To make this possible:
+- The generic type of the `effect` method needs to be set to `void`.
+- The name of the source stream doesn't matter: `trigger$`, `source$` or `$` are good names.
```

Comment:
```suggestion
- The name of the source stream doesn't matter: `trigger$`, `source$`, or `$` are good names.
```

Reply:
you really want to add a coma before or ?

Reply:
It's called [serial comma](https://en.wikipedia.org/wiki/Serial_comma). So far I thought that's a rule in English grammar, but it's a preference. 😅 So you can ignore this comment.